roles/sft-server/templates/sftd.vhost.conf.j2

```diff
@@ -1,3 +1,7 @@
+limit_req_zone "$binary_remote_addr$request_uri" zone=reqs_per_addr:12m rate=10r/s;
```

Comment:
Isn't 10r/s too much per IP? Note current wire.com limit for unauthenticated requests is 5r/m.

Considering these requests will all be unauthenticated (from nginx perspective) for the time being, probably some middle ground between those values would make sense. We can also stick with this value for now and tighten it when needed.

Reply:
Shouldn't this be `"$binary_remote_addr$uri"` ? `uri` is [normalized](https://nginx.org/en/docs/http/ngx_http_core_module.html#var_uri), whereas request_uri [isn't](https://nginx.org/en/docs/http/ngx_http_core_module.html#var_request_uri). Probably you could sidestep the rate limiting here when using the same URL but written differently (e.g. using `/../` elements).

Or did you have a motivation to change from `uri` to `request_uri` that isn't documented?

Reply:
Yes, you are right. I looked at it from a different angle from which `request_uri` made sense to me. 

About the requests per seconds. I picked the numbers for *authenticated users* looking at it from that role, because it's the one a client would we in when calling. But as you said, I think, it's something worth iterating on when we have more data.